pkg/properties/propertiesGet_test.go

```diff
@@ -364,22 +514,270 @@ func TestValidNamespaceAndNoMatchingSuffixReturnsOk(t *testing.T) {
+	apiServerUrl := server.URL
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `namespace      name               value
+validNamespace extra.infix1.extra infixVal1
+validNamespace extra.infix3.extra infixVal3
+
+Total:2
+`
+	//When
+	err := GetProperties(namespace, name, prefix, suffix, infix, apiServerUrl, propertiesOutputFormat, console)
+
+	//Then
+	assert.Nil(t, err)
+	assert.Equal(t, expectedOutput, console.ReadText())
+}
+
+func TestValidNamespaceWithMatchingPrefixAndMultipleInfixReturnsOk(t *testing.T) {
+	//Given...
+	namespace := "validNamespace"
+	name := ""
+	prefix := "aPrefix"
+	suffix := ""
+	infix := "infix1,blahblah,infix3"
+	propertiesOutputFormat := "summary"
+
+	server := NewPropertiesServletMock(t)
+	apiServerUrl := server.URL
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `namespace      name                 value
+validNamespace aPrefix.infix1.extra prefixMultInfixVal1
+validNamespace aPrefixinfix3.extra  prefixMultInfixVal3
+
+Total:2
+`
+	//When
+	err := GetProperties(namespace, name, prefix, suffix, infix, apiServerUrl, propertiesOutputFormat, console)
+
+	//Then
+	assert.Nil(t, err)
+	assert.Equal(t, expectedOutput, console.ReadText())
+}
+
+func TestValidNamespaceWithMatchingSuffixAndMultipleInfixReturnsOk(t *testing.T) {
+	//Given...
+	namespace := "validNamespace"
+	name := ""
+	prefix := ""
+	suffix := "aSuffix"
+	infix := "infix1,blahblah,infix3"
+	propertiesOutputFormat := "summary"
+
+	server := NewPropertiesServletMock(t)
+	apiServerUrl := server.URL
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `namespace      name                 value
+validNamespace extra.infix1.aSuffix suffixMultInfixVal1
+validNamespace extra.infix3.aSuffix suffixMultInfixVal3
+
+Total:2
+`
+	//When
+	err := GetProperties(namespace, name, prefix, suffix, infix, apiServerUrl, propertiesOutputFormat, console)
+
+	//Then
+	assert.Nil(t, err)
+	assert.Equal(t, expectedOutput, console.ReadText())
+}
+
+func TestValidNamespaceWithMatchingPrefixAndSuffixAndMultipleInfixReturnsOk(t *testing.T) {
+	//Given...
+	namespace := "validNamespace"
+	name := ""
+	prefix := "aPrefix"
+	suffix := "aSuffix"
+	infix := "infix1,blahblah,infix3,extra"
+	propertiesOutputFormat := "summary"
+
+	server := NewPropertiesServletMock(t)
+	apiServerUrl := server.URL
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `namespace      name                   value
+validNamespace aPrefix.infix1.aSuffix prefixsuffixMultInfixVal1
+validNamespace aPrefix.infix3aSuffix  prefixsuffixMultInfixVal3
+
+Total:2
+`
 	//When
-	err := GetProperties(namespace, name, prefix, suffix, apiServerUrl, propertiesOutputFormat, console)
+	err := GetProperties(namespace, name, prefix, suffix, infix, apiServerUrl, propertiesOutputFormat, console)
 
 	//Then
 	assert.Nil(t, err)
 	assert.Equal(t, expectedOutput, console.ReadText())
 }
 
```

Comment:
These tests appear to be testing the API functionality not the CLI.
You should be able to delete them without impacting your testing coverage